lisa/operating_system.py

```diff
@@ -225,11 +290,32 @@ class Fedora(Linux):
     def name_pattern(cls) -> Pattern[str]:
         return re.compile("^Fedora|fedora$")
 
-    def _install_packages(self, packages: Union[List[str]]) -> None:
-        self._node.execute(
-            f"dnf install -y {' '.join(packages)}",
-            sudo=True,
+    def _install_packages(
+        self, packages: Union[List[str]], signed: bool = True
+    ) -> ExecutableResult:
+        command = f"dnf install -y {' '.join(packages)}"
+        if not signed:
+            command.__add__(" --nogpgcheck")
+        return self._node.execute(command, sudo=True)
+
+    def _get_os_version(self) -> OsVersion:
+        os_version = OsVersion("")
+        cmd_result = self._node.execute(
+            cmd="cat /etc/redhat-release", no_error_log=True
```

Comment:
From the name, it should be written in RedHat class, not Fedora class.

Reply:
> From the name, it should be written in RedHat class, not Fedora class.

hmm /etc/redhat-release can give the information for Fedora distros too.
But let me check more on /etc/os-release, if that gives info for most of the distros then I will add that first in the Linux class.

Reply:
> > From the name, it should be written in RedHat class, not Fedora class.
> 
> hmm /etc/redhat-release can give the information for Fedora distros too.
> But let me check more on /etc/os-release, if that gives info for most of the distros then I will add that first in the Linux class.

Done